robottelo/constants/__init__.py

```diff
@@ -1804,11 +1804,11 @@
 BOOKMARK_ENTITIES = [
     {'name': 'ActivationKey', 'controller': 'katello_activation_keys'},
     {'name': 'Dashboard', 'controller': 'dashboard', 'skip_for_ui': True},
-    {'name': 'Fact', 'controller': 'fact_values', 'skip_for_ui': True},
     {'name': 'Audit', 'controller': 'audits', 'skip_for_ui': True},
     {'name': 'Report', 'controller': 'config_reports', 'skip_for_ui': True},
     {'name': 'Task', 'controller': 'foreman_tasks_tasks', 'skip_for_ui': True},
-    {'name': 'Subscriptions', 'controller': 'katello_subscriptions', 'skip_for_ui': True},
+    # TODO Load manifest for the test_positive_end_to_end from the ui/test_bookmarks.py
+    # {'name': 'Subscriptions', 'controller': 'subscriptions', 'skip_for_ui': True},
```

Comment:
@jaryn did you intend to leave these lines in as comments?

Reply:
I guess I should rather create a Jira ticket.